.github/workflows/ci.yml

```diff
@@ -41,8 +41,8 @@ jobs:
       run: vendor/bin/phpcs
     - name: lint twig
       run: bin/console lint:twig templates custom
-    - name: security check
-      run: bin/console security:check
+    - name: Local PHP Security Checker
+      uses: docker://pplotka/local-php-security-checker-github-actions
```

Comment:
Oooh, this scares me a bit. Unversioned random docker container? Is there an official action we could use instead?

Reply:
unfortunately, there's no official support for this yet.

Reply:
so, shall i yank this then?

Reply:
per our out-of-band convo, i yanked the gh action.